ibm_catalog.json

```diff
@@ -53,32 +58,54 @@
           "iam_permissions": [
             {
               "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "service_name": "all-account-management-services"
+              "service_name": "kms",
+              "notes": "[Optional] Required if KMS encryption is enabled and Key protect is used for encryption."
             },
             {
               "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "service_name": "lakehouse"
+              "service_name": "hs-crypto",
+              "notes": "[Optional] Required if Hyper Protect Crypto Services (HPCS) encryption is enabled and is used for encryption."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "All Account Management services",
+              "notes": "[Optional] Required when setting up foundational IBM Cloud account components such as IAM settings, trusted profiles, access groups, and resource groups."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "lakehouse",
+              "notes": "Required for creating a new instance of watsonx.data"
             }
           ],
           "architecture": {
             "features": [
               {
-                "title": " Creates and configures an instance of watsonx.data",
-                "description": "This architecture creates and configures an instance of IBM watsonx.data. It accepts or creates a resource group."
+                "title": "watsonx.data instance",
+                "description": "Creates and configures an instance of watsonx.data."
+              },
+              {
+                "title": "KMS Keys and Key Rings",
+                "description": "Supports the creation of Keys and Key Rings for encrypting watsonx.data instances."
               }
             ],
             "diagrams": [
               {
                 "diagram": {
-                  "caption": "watsonx.data on IBM Cloud",
+                  "caption": "IBM watsonx.data architecture",
```

Comment:
Please revert the caption, as such we are really not talking about watsonx.data architecture.